index.js

```diff
@@ -1,2 +1,3 @@
 <h1> Ejercicio 1</h1>
-<h2> Aprender a hacer commit con github</h2>
\ No newline at end of file
+<h2> Aprender a hacer commit con github</h2>
+<p>Hola mundo</p>
```

Comment:
No me gusta, quiero exclamación
```suggestion
<p>Hola mundo!!</p>
```